src/state.c

```diff
@@ -2048,6 +2059,7 @@ parseDNSAnswer(char *buf, size_t len, cJSON *json, cJSON *addrs, int first)
 
             if (scope_ns_parserr(&handle, ns_s_an, i, &rr) == -1) {
                 scopeLogError("ERROR:parse rr");
+                dnsSecurity("nil", "illegal DNS response can't be parsed");
```

Comment:
i used "nil" here as the dns name. do you have any better ideas?

Reply:
Seems like a good approach. We don't have a result in several cases.